api/v0/serializers.py

```diff
@@ -1,23 +1,84 @@
 import json
 
+# FIXME: serializers.ValidationError not exceptions.ValidationError
```

Comment:
Only for my new tests, this should also be the case for the others according to DRF docs. I wanted to get the tests passing as they were first.